examples/java-agent/src/main/java/example/HelloWorld.java

```diff
@@ -17,9 +17,16 @@
 package example;
 
 import static spark.Spark.get;
+import static spark.Spark.port;
 
 public class HelloWorld {
   public static void main(String[] args) {
-    get("/hello", (req, res) -> "Hello World");
+    // Allow use with Cloud Run which requires listening on the value in PORT
+    String portEnv = System.getenv("PORT");
+    if(portEnv != null) {
```

Comment:
`if(portEnv` --> `if (portEnv`, or it's just not in Google style?

Reply:
Huh, I thought we had a validator to break the build.

Reply:
I don't think that runs on the example projects.